spec/classes/rabbitmq_spec.rb

```diff
@@ -1161,6 +1161,101 @@
         end
       end
 
+      describe 'ssl options with ssl_crl_check enabled' do
+        let(:params) do
+          { ssl: true,
+            ssl_port: 3141,
+            ssl_cacert: '/path/to/cacert',
+            ssl_cert: '/path/to/cert',
+            ssl_key: '/path/to/key',
+            ssl_crl_check: 'true'
+        end
+
+        it 'sets ssl crl check setting to specified value' do
+          is_expected.to contain_file('rabbitmq.config').with_content(%r{crl_check,true})
+        end
+      end
+
+      describe 'ssl options with ssl_crl_check and ssl_crl_hash_cache enabled' do
+        let(:params) do
+          { ssl: true,
+            ssl_port: 3141,
+            ssl_cacert: '/path/to/cacert',
+            ssl_cert: '/path/to/cert',
+            ssl_key: '/path/to/key',
+            ssl_crl_cache_hash_dir: '/path/to/crl_cache/dir'
+        end
+
+        it 'sets ssl crl check setting to specified value' do
+          is_expected.to contain_file('rabbitmq.config').with_content(%r{crl_check,true})
+          is_expected.to contain_file('rabbitmq.config').with_content(%r{\[crl_cache,[[:space:]]+{ssl_crl_hash_dir,[[:space:]]+{internal,[[:space:]]+[{dir, "/path/to/crl_cache/dir"}]}}\]})
```

Comment:
nit, but I think `\s` might be more consistent than `[[:space:]]` with the rest of the file possibly?

Reply:
fixed.